tests/test_wave_01.py

```diff
@@ -131,13 +132,14 @@ def test_update_task_not_found(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
     # *****************************************************************
     # **Complete test with assertion about response body***************
     # *****************************************************************
+    assert response_body == {"details": "Invalid Data"}
```

Comment:
If you changed your response like how I recommended above, this test would no longer pass, which is a good thing! You'd then need to update it to reflect the new details message.